elements/pfe-dropdown/src/pfe-dropdown.scss

```diff
@@ -0,0 +1,160 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: dropdown;
+
+$variables: (
+  toggle: (
+    MinWidth: calc(#{pfe-var(container-spacer)} * 2),
+    MaxWidth: 100%,
+    SpacingVertical: calc(#{pfe-var(container-spacer)} * 0.5),
+    SpacingHorizontal: calc(#{pfe-var(container-spacer)} * 0.5),
+    FontSize: #{pfe-var(font-size)},
+    FontWeight: 400,
+    LineHeight: #{pfe-var(line-height)},
+    Color: #{pfe-color(text)},
+    BackgroundColor: transparent,
+    Border: none,
+    BorderBottomColor--before: #8a8d90,
+    BorderBottomColor--before--hover: #{pfe-color(link)},
+    BorderWidth--before: #{pfe-var(ui--border-width)},
+    BorderWidth--before--hover: calc(#{pfe-var(ui--border-width)} * 2),
+    BorderTopColor--before: #ededed,
+    BorderRightColor--before: #ededed,
+    BorderLeftColor--before: #ededed,
+    BorderStyle--before: solid
+  ),
+  menu: (
+    SpacingVertical: calc(#{pfe-var(container-spacer)} * 0.5),
+    SpacingHorizontal: 0,
+    Margin: 0,
+    Background: #{pfe-color(surface--lightest)},
+    BorderWidth: #{pfe-var(ui--border-width)},
+    BorderStyle: #{pfe-var(ui--border-style)},
+    BorderColor: transparent,
+    BoxShadow: #{pfe-var(box-shadow--sm)},
+    MinWidth: 100%
+  )
+);
+
+:host {
+  @include pfe-print-variables($variables);
+  position: relative;
+  display: inline-block;
+  max-width: 100%;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([is_disabled]) {
+  .pfe-dropdown {
+    &__container {
+      pointer-events: none;
+    }
+    &__toggle {
+      --pfe-dropdown__toggle--BackgroundColor: #ededed;
```

Comment:
$pfe-color--gray-100 is #ececec, could we reference that variable?

https://www.sassmeister.com/gist/4f095523377d120792295e321c73e01c